src/types/user.types.ts

```diff
@@ -38,10 +38,10 @@ export const userResponse = Type.Object({
 export type UserResponse = Static<typeof userResponse>;
 
 export const createUserRequestBody = Type.Object({
-  email: Type.String(),
-  forename: Type.String(),
-  surname: Type.String(),
-  password: Type.String(),
+  email: Type.String({ minLength: 3, maxLength: 254, pattern: '^.+@.+$' }),
```

Comment:
```suggestion
  email: Type.String({ minLength: 3, maxLength: 254, format: 'email' }),
```

Reply:
Regexisi näyttää hitusen liian simppeliltä muutenkin 
https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression

Reply:
Tässä vielä formateista -> https://ajv.js.org/guide/formats.html

Reply:
Kehitellään hieman tiukempi seula

Reply:
Pitäen mielessä tän:
https://stackoverflow.com/questions/48055431/can-it-cause-harm-to-validate-email-addresses-with-a-regex

Reply:
Korjattu: https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/pull/20/commits/0f5d6e21c660086fbae9f5c2ef8251d8e2376e9c